openmeetings-web/src/main/java/org/apache/openmeetings/web/admin/connection/dto/ConnectionListItem.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I see no value in this class :(

Reply:
Wicket will blow up if you return the KStream in the bean.

Reply:
You simply can't put that object ref into the ModelObject.

Reply:
I had to wrap KStream and Client into a new object to have a single list that I can iterate over in the list component.